src/components/Button.js

```diff
@@ -47,6 +47,9 @@ const propTypes = {
     /** Call the onPress function when Enter key is pressed */
     pressOnEnter: PropTypes.bool,
 
+    /** The priority to assign the enter key event listener */
+    enterKeyEventListenerPriority: PropTypes.number,
```

Comment:
So for some background, the `Button` prop has a `pressOnEnter` parameter. If that parameter is passed, then pressing the `Enter` key will effectively press the button.

However, since we're creating a global queue of keyboard event listeners, there are times when we might have multiple handlers for the same key. And in some cases, we want a new handler to get a lower priority than an existing handler, and that's what this prop controls. Let me explain with an example:

1. We are on the New Group Page, and we have an event handler for the `Event` key that will select a result from the list the we have highlighted. So symbolically our handlers look like this:
    
    ```
    {
        'Enter': [ToggleGroupParticipant]
    }
    ```

1. Once we have a selected result, the `Create new group` button appears. Because the `Button` prop has the `pressOnEnter` parameter, it sets up a new handler for the `Enter` key event. If we don't assign any specific priority, our new handlers will look something like this:

    ```
    {
        'Enter': [CreateNewGroup, ToggleGroupParticipant],
    }
    ```

    So `CreateNewGroup` will be executed _before_ `ToggleGroupParticipant`. As a result, if we highlight another unselected option and press `Enter`, we'll create a new group and leave the page.
1. However, if we assign a priority of `1` to the new listener, it would look like this instead:

    ```
    {
        'Enter': [
            // Note: This callback can choose to allow the `Enter` keydown event to "bubble", allowing later callbacks to execute after it
            ToggleGroupParticipant,
            CreateNewGroup
        ],
    }
    ```
    
    So instead, if we press `Enter` on the NewGroupPage with the `Create new group` button visible, we'll _first_ try to toggle a group participant, and if one's not highlighted (i.e: we're focused on the search bar), then we allow the `Enter` key event to bubble and the new group is created.
    

That was a long-winded answer ... does that clarify why I added this prop?

Reply:
Let's add a comment explaining that 0 is the highest priority.